src/Path/Pretty.hs

```diff
@@ -18,49 +38,43 @@ module Path.Pretty
 
 import Control.Arrow ((***))
 import Control.Monad.IO.Class
-import Data.Foldable (fold)
-import Data.List (isSuffixOf)
 import Path.Span
 import System.Console.Terminal.Size as Size
 import System.IO (stdout)
 import System.IO.Unsafe
-import Text.PrettyPrint.ANSI.Leijen as PP hiding ((<$>), bool, column, empty, putDoc)
+import qualified Data.Text as Text
+import Data.Text.Prettyprint.Doc as PP hiding (Doc, Pretty (..), column)
+import qualified Data.Text.Prettyprint.Doc as PP
+import Data.Text.Prettyprint.Doc.Internal (unsafeTextWithoutNewlines)
+import Data.Text.Prettyprint.Doc.Render.Terminal (AnsiStyle, Color (..), color, colorDull)
+import qualified Data.Text.Prettyprint.Doc.Render.Terminal as ANSI
 
-prettyPrint :: (Pretty a, MonadIO m) => a -> m ()
-prettyPrint = putDoc . pretty
+type Doc = PP.Doc AnsiStyle
```

Comment:
`prettyprinter`’s `Doc` type is parameterized by the type of annotations, which we currently always want to specialize to `AnsiStyle`.